dist/middleware.js

```diff
@@ -112,7 +112,7 @@ exports.default = _assign2.default || function (target) {
 
 
 
-var _jsxFileName = '/home/ajit/Documents/projects/contributions/gutenberg-fields-middleware/middleware/components/field/index.js';
+var _jsxFileName = '/Users/yahilmadakiya/Local Sites/wp50/app/public/wp-content/plugins/gutenberg-fields-middleware/middleware/components/field/index.js';
```

Comment:
Not a blocker but this shouldn't include name, you may fix it in another PR.

Reply:
Sure.